safegraph/delphi_safegraph/process.py

```diff
@@ -1,13 +1,51 @@
-import covidcast
+"""Internal functions for creating Safegraph indicator."""
+import datetime
+from typing import List
 import numpy as np
 import pandas as pd
+import covidcast
 
 from .constants import HOME_DWELL, COMPLETELY_HOME, FULL_TIME_WORK, PART_TIME_WORK
-from .geo import FIPS_TO_STATE
+from .geo import FIPS_TO_STATE, VALID_GEO_RESOLUTIONS
 
 # Magic number for modular arithmetic; CBG -> FIPS
 MOD = 10000000
 
+
+def validate(df):
```

Comment:
Is this (and below) something that can be shared across multiple indicators?

Reply:
Possibly.  I'm working on a survey of the indicators to see what sorts of libraries we can or would want to refactor out.  Would prefer to do in a followon PR.

Reply:
+1 that's exactly what I was wondering. Agree definitely not for this PR.